src/components/Divider/MucDivider.vue

```diff
@@ -1,9 +1,3 @@
 <template>
-  <hr class="divider-border" />
+  <div class="muc-divider" />
```

Comment:
We need to help screenreaders with the empty div ... [Comment](https://github.com/it-at-m/muc-patternlab-vue/issues/183#issuecomment-2428720690)

Reply:
In which way do we need to help screenreaders with this?

Most screenreaders ignore hr-Tags all together and empty divs are handled the same way.

Reply:
The `hr` tag define when a new section starts or ends. This tag will is will be read as a separator.
We can postpone this into another issue